lazyflow/metaDict.py

```diff
@@ -40,6 +40,9 @@ def __init__(self, other=None, *args, **kwargs):
             # are ready
             self._ready = False
 
+        if not 'has_mask' in self:
+            self.has_mask = False
+
```

Comment:
I'd still prefer to change it.  It isn't obvious here, but `_ready` is a very special case -- it's not considered part of this class's "public" attributes.  But the new `has_mask` attribute isn't special -- it is very analogous to other fields like `prefer_2d`, among others.